vizro-core/tests/unit/vizro/actions/test_parameter_action.py

```diff
@@ -137,6 +174,35 @@ def test_one_parameter_one_target(
         assert result["scatter_chart"] == target_scatter_parameter_y
         assert "box_chart" not in result
 
+    @pytest.mark.parametrize(
+        "callback_context_parameter_hover_data, target_scatter_parameter_hover_data",
+        [(["NONE", "pop"], ["pop"]), (["NONE"], [None])],
+        indirect=True,
+    )
+    def test_one_parameter_one_target_NONE_list(
+        self,
+        callback_context_parameter_hover_data,
+        target_scatter_parameter_hover_data,
+    ):
+        # Creating and adding a Parameter object to the existing Page
+        y_parameter = vm.Parameter(
+            id="test_parameter",
+            targets=["scatter_chart.hover_data"],
+            selector=vm.Dropdown(
+                id="hover_data_parameter", multi=True, options=["None", "lifeExp", "pop", "gdpPercap"], value=["None"]
```

Comment:
No, you are correct, corrected now!

Reply:
Was the test definitely testing the right thing then? Because It seemed to pass before when you had this as `"None"`.

Reply:
Yes, because it didn't matter in the test cases used. The callback context faked that the user would select `"NONE"`, at which point the original options or value do not matter.

Having introduced `"ALL"` this changed, and the error showed, so I fixed it :)